packages/compartment-mapper/test/snapshots/test-infer-exports.js.md

```diff
@@ -0,0 +1,140 @@
+# Snapshot report for `test/test-infer-exports.js`
+
+The actual snapshot is saved in `test-infer-exports.js.snap`.
+
+Generated by [AVA](https://avajs.dev).
+
+## infer-exports for @floating-ui/dom
+
+> Snapshot 1
+
+    {
+      exports: {
+        '@floating-ui/dom': './dist/floating-ui.dom.esm.js',
+        '@floating-ui/dom/package.json': './package.json',
+      },
+      types: {
+        './dist/floating-ui.dom.esm.js': 'mjs',
+      },
+    }
+
+## infer-exports for @noble/hashes
+
+> Snapshot 1
+
+    {
+      exports: {
+        '@noble/hashes/_sha2': './_sha2.js',
+        '@noble/hashes/blake2b': './blake2b.js',
+        '@noble/hashes/blake2b.d.ts': 'blake2b.d.ts',
```

Comment:
Before I merged origin/master these were `./blake2b.d.ts` and now they're not relativized.
When I make a change to relativize them, I get a test failure elsewhere

```
retained › archives only contain compartments retained by modules

  Returned promise rejected with:

  Error {
    message: 'Cannot find file for internal module "./mk2" (with candidates "./mk2", "./mk2.js", "./mk2/index.js") in package file:///home/naugtur/work/metamask/endo/packages/compartment-mapper/test/fixtures-retained/node_modules/mk1/',
  }

  › file://test/test-retained.js:16:11

```

@kriskowal which is correct? I thought they should all be relativized

Reply:
I recently made a change that makes it possible for a package to reëxport an external dependency, e.g., `"exports": {".": "fs"}`. It doesn’t seem likely to me that the `exports` feature intentionally limits that possibility by relativizing everything, but maybe it does. I think I will have to depend on your research to tell.

Reply:
These are not exports in package.json, these are results of running inferExports and are a flat key-value with values to use by compartment-mapper. 
I was hoping to get a "no, these won't work if not relativized" or "this is fine" ;)

Reply:
No misunderstanding. These will not work if not relativized. That means you would have to revert https://github.com/endojs/endo/commit/5a8a893b9f9d4375661fae597c158a0bbf258785 to get this to work. That makes me sad, but is consistent with the usage in `@noble/hashes` and therefore indicates a mistake in the design of `"exports"` that we can’t fix.

Reply:
@kriskowal done. I even persisted some of the sadness from yesterday's meeting in a comment.
Let's merge this ;)